diagnostic.md

```diff
@@ -23,7 +27,9 @@ Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
 <!-- your answer starts here -->
-
+$("li").parent()
+$("ol").parent()
+$("body").children
```

Comment:
Almost there. Here's one way to do it:

`$('li').parent().parent().find('form')`